src/config.rs

```diff
@@ -149,6 +154,7 @@ impl Default for DbOptions {
             l0_max_ssts: 8,
             compactor_options: Some(CompactorOptions::default()),
             compression_codec: None,
+            disk_cache_root_folder: None,
```

Comment:
Should part size be a config as well?

Reply:
yeah it will, i'd like to expose the config in another pr with a better documentation about it